fuzzers/005-tilegrid/pcie_int_interface/generate.tcl

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2017-2020  The Project X-Ray Authors
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+source "$::env(XRAY_DIR)/utils/utils.tcl"
+
+proc parse_csv {} {
+    set fp [open "params.csv"]
+    set file_data [read $fp]
+    close $fp
+
+    set file_data [split $file_data "\n"]
+
+    set params_map [dict create]
+
+    set is_first_line true
+    foreach line $file_data {
+        if { $is_first_line } {
+            set is_first_line false
+            continue
+        }
+
+        set parts [split $line ","]
+
+        dict lappend params_map [lindex $parts 0] [lindex $parts 1]
+    }
+
+    puts $params_map
+    return $params_map
+}
+
+
+proc route_through_delay {} {
+    set params_map [parse_csv]
+    set nets [get_nets]
+
+    dict for { key value } $params_map {
+        if { $value == 0 } {
+            continue
+        }
+
+        if { $key == "" } {
+            puts "Dictionary key is incorrect, continuing"
```

Comment:
When would this ever happen?  Rather than skip a bad key, why not fix the reason it exists?

Reply:
Fixed the params.csv parsing to avoid having to check the dictionary key